settings.gradle

```diff
@@ -0,0 +1,2 @@
+rootProject.name = 'buisenessProject'
```

Comment:
why root project is businessProject instead of business? 
you named your project like businessProject?